nservicebus/hosting/docker-host/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Docker Container Host
+summary: Take advantage of process isolation by hosting endpoints in Docker containers
+tags:
+ - Hosting
+related:
+ - nservicebus/dotnet-templates
+ - nservicebus/licensing
+ - samples/hosting/docker
+reviewed: 2018-02-28
+---
+
+Docker containers provide the ability to deploy endpoints in a self contained manner. To create an endpoint to host in a Docker container, download and install the [Docker endpoint template](/nservicebus/dotnet-templates.md), and create a new project using `dotnet new nsbdockerendpt`. The project that is created will have the required endpoint setup infrastructure in addition to the needed `DOCKERFILE` to create and deploy a container hosting one endpoint.
```

Comment:
self-contained

Reply:
Also, not sure if a comma is actually needed before  `and create a new project using`